packages/webapp/src/routes/developer/confirm-off-site-gain.js

```diff
@@ -0,0 +1,87 @@
+import constants from '../../utils/constants.js'
+import { deleteBlobFromContainers } from '../../utils/azure-storage.js'
+import { logger } from 'defra-logging-facade'
+
+const href = '#offsite-details-checked-yes'
+const handlers = {
+  get: (request, h) => {
+    const context = getContext(request)
+    return h.view(constants.views.DEVELOPER_CONFIRM_OFF_SITE_GAIN, {
```

Comment:
can't these spread operators just be replaced by passing in context to the view eg `return h.view(constants.views.DEVELOPER_OFF_SITE_GAIN, context)` ?